File: Tarea/Ejercicio09.py
#Es el ejercicio 3.11
def ejercicio09():
  #Inicio
  print("Buenos dias")
  #Introducccion de datos
  años=int(input("¿Cuantos años lleva trabajando?:"))
  #Proceso
  bono=0.0
  mensaje=""
  cantidad=años*100
  if años>=1 and años<=5:
    bono=cantidad
  elif años>5:
    bono=1000
  else:
    bono="Datos nulos, vuelva a intertarlo"

  
  #Datos de salida
  print("Su bono es de:", bono)
ejercicio09()

   
  

File: Tarea02/Ejercicio09.java
//Este e el ejercicio 4.8
import java.util.Scanner;
class Ejercicio09{
    static Scanner teclado=new Scanner(System.in);
    static void ejercicio09FTT(){ 
        //Inicio
        System.out.println("Hola, buenos dias");
        //Introduccion de datos
        int cantidad=1, articulos=1;
        double precio, total=0; 
        System.out.println("Cuantos articulos comprara: ");
        cantidad=teclado.nextInt();
        
        while(articulos<=cantidad){ 
        System.out.println("Ingrese el costo de produto numero "+articulos);
        precio=teclado.nextDouble();
            if(precio>=200){
                precio=precio-(precio*0.15);
            }else if(precio>100 && precio<200){
                precio=precio-(precio*0.12);
            }else{
                precio=precio-(precio*0.10);
            }
            total=total+precio;
            articulos++;
            
        }
        System.out.println("El costo total de sus "+cantidad+" objetos comprados es de compra es de: "+total);

        }
    public static void main(String[] args) {
    ejercicio09FTT();
    }
}


